### DISCUSSION TITLE
Can't install SSL certs

### DISCUSSION THREAD

**Discussion by bertalting** [telekom-security/tpotce#1140] [Q&A]
I can't figure it out how to install my own SSL certs (crt / key). 
Can't find it either on the Wiki Pages.

Someone tips / clues?

**Comment by Ma5onic** [comment]
Check out these for instructions/clues:

https://github.com/telekom-security/tpotce/issues/437#issuecomment-521623873

https://www.youtube.com/watch?v=dhZqXvZmtiM
